scale_build/clean.py

```
@@ -25,6 +25,10 @@ def clean_packages():
             os.makedirs(path)
 
 
+def clean_diversions():
+    shutil.rmtree(DIVERT_DIR, ignore_errors=True)
```

I think this is failing silently. Will have to investigate.